.github/workflows/link-checker.yml

```diff
@@ -4,6 +4,8 @@ on:
       - main
   pull_request:
     types: [opened, synchronize, reopened]
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
#813 #817 #789

Reply:
> #813 #817 #789

Someone who is an org owner should block this annoying bot.

Reply:
done.